chart/openfaas/templates/queueworker-dep.yaml

```
@@ -60,6 +58,10 @@ spec:
         - name: faas_function_suffix
           value: ".{{ .Values.functionNamespace }}.svc.{{ .Values.kubernetesDNSDomain }}"
         {{- end }}
+        {{- if .Values.queueWorker.maxInflight }}
+        - name: max_inflight
+          value: ".{{ .Values.queueWorker.maxInflight }}"
```

Is this `.` prefix correct?
```suggestion
          value: "{{ .Values.queueWorker.maxInflight }}"
```

---

No that's a good catch thanks